curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/*
+ * Project: curve
+ * Created Date: Thur May 27 2021
+ * Author: xuchaojie
+ */
+#include "curvefs/src/client/curve_fuse_op.h"
+#include "curvefs/src/client/fuse_client.h"
+
+using ::curvefs::client::FuseClient;
+
+static FuseClient *g_ClientInstance = nullptr;
+
+extern "C" {
```

Comment:
this `extern "C"` is redundant, because those codes in here aren't compiled as a shared library that is provided to the c program.

Reply:
done